### PR TITLE
{lib}[GCCcore/10.3.0] filevercmp v20191210

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node163.pri.viking.alces.network - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8e98917c3b267758cf038b2010c2350f for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/e7614f79fab807ae80c2ebec39a22592 for a full test report.

File: easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+skipsteps = ['configure']
+
+installopts = 'DESTDIR="" PREFIX=%(installdir)s '
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s', 'lib/libfilevercmp.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.3.0.eb:32]: Could we add a sanity command here, which just passes two dummy strings to the tool?

Comment (verdurin) [reply easybuild/easyconfigs/f/filevercmp/filevercmp-20191210-GCCcore-10.3.0.eb:32]: @bedroge Done.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/35216ec49e4de9a18c6ad294b24b2a4b for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/9fd9c93b30c0bdfd69a0c4af6b2c5f64 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @verdurin!

**Pull request merged**